### ISSUE TITLE
change apps settings behavior

### ISSUE THREAD

**Issue by mytvpro** [GhostWriters/DockSTARTer#613]
possible to exit the script after setting the one(s) you want without have to go through each app until the end? maybe, change setting: "previous" "yes" "no" "exit"?

**Comment by mytvpro** [comment]
or can cntrl+c be used here safely?

**Comment by nemchik** [comment]
ctrl+c can be safely used anywhere in the GUI and anything you have done on any previous screen will be saved (the current screen would not be).

You may be more interested in opening `~/.docker/compose/.env` and editing your settings directly. The GUI is just a fancy guided editor for the .env file. Full config setup is always recommended for the first run, but after that you're likely fine to just do what you need to with the .env file. After making any changes to the .env file you would need to run `ds -c` to make the changes affect your system. DS will recreate only the containers affected by the changes you made or those who have an updated image available on your system. Your system would only have updated images if you pull them using `ds -c pull` or if an app like ouroboros or watchtower pulled them for you (but those apps should also be recreating your containers with the new images for you).

You can see a full list of what DS can do via the command line by running `ds -h`. All of the options you will see in the usage information are CLI only and do not involve the GUI.

**Comment by mytvpro** [comment]
That's very helpful thank you

**Comment by mytvpro** [comment]
> After making any changes to the .env file you would need to run `ds -c` to make the changes affect your system. 

`ds -c' is reinstalling docker-compose each time which is failing on my ubuntu 18.10 (maybe doing this because it was installed independently due to this failure?) so i guess not a viable option for my to edit .env manually. unless there is a way to do run 'ds -c' and bypass this reinstall of docker-compose?

**Comment by mytvpro** [comment]
docker-compose 1.24.0
@ulyssessouza ulyssessouza released this 2 hours ago

that's why i updated manually again

**Comment by nemchik** [comment]
Really no need to ping the dev of compose over something this project is doing.

Ubuntu 18.10 is explicitly mentioned in the readme as known to have issues. I recommend you use 18.04.

What the script is doing is running `docker --version` and as of writing this response it expects `18.09.4` or greater. If the version installed is less it will run `curl -fsSL get.docker.com | sh`

Then the script runs `docker-compose --version` and as of writing this response it expects `1.24.0` or greater. If the version installed is less it will run a number of commands, but the most important is `pip install -IUq docker-compose`

Sometimes shortly after a release is tagged in their repo it can take a bit to be available for download. For example I ran the update right when I got the email notification of a new version but i didn't get the update installed until a few minutes ago. It usually doesn't take very long though.
